src/lib/mongoBackend/mongoQueryTypes.cpp

```diff
@@ -157,7 +157,8 @@ HttpStatusCode mongoEntityTypes
   const std::string&                   tenant,
   const std::vector<std::string>&      servicePathV,
   std::map<std::string, std::string>&  uriParams,
-  const std::string&                   apiVersion
+  const std::string&                   apiVersion,
+  unsigned int*                        totalTypes
```

Comment:
To indicate that totalTypes is a pointer (and that might get modified inside the function), we usually add a 'P' at the end of the variable name - totalTypesP.

This we should also add to the style guide, as SHOULD.